### ISSUE TITLE
Insert the help center icon also as a favicon

### ISSUE THREAD

**Issue by flaviomoceri** [chatwoot/chatwoot#8571]
### Is your feature or enhancement related to a problem? Please describe.

I'm hosting chatwoot, I'd like the icon I inserted in the help center to also be set as a favicon

### Describe the solution you'd like

.

**Comment by pranavrajs** [comment]
I'm sorry, I don't understand the request. Can you please provide more information on this? Do you mean to the logo you uploaded to the help center as the favicon on the title bar?

**Comment by flaviomoceri** [comment]
Yes exactly, I would like to be able to insert a favicon in the help center

**Comment by pranavrajs** [comment]
Makes sense, we will take this up as feature in the next releases.

**Comment by flaviomoceri** [comment]
Also being able to have Live Chat support in the help center would be great

**Comment by stonecharioteer** [comment]
This issue is being closed due to lack of activity for over a year. [For more details, visit this discussion.](https://github.com/orgs/chatwoot/discussions/12810)
